app/assets/stylesheets/welcome.css.scss

```diff
@@ -0,0 +1,26 @@
+#banner {
+	background-image: url(image_path('welcome.jpg'));
+	background-size: cover;
+	background-repeat: no-repeat;
+	background-position: top center;
+	width: 100%;
+	min-height: 500px;
+	display: table;
+	.banner_content {
+		display: table-cell;
+		vertical-align: middle;
+		text-align: center;
+		h1 {
+			font-family: 'Amaranth', sans-serif;
+			font-style: italic;
+			color: white;
+			font-size: 3rem;
+			margin: 0;
+		}
+		p {
+			font-size: 1.2rem;
+			margin: 0 0 2.5rem 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0 0 2.5rem` instead of `0 0 2.5rem 0`